src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke.task;
+
+/**
+ * A type of Task, which is to be done at somewhere.
+ */
+public class Event extends Task{
+    protected String at;
+
```

Comment:
Same as the deadline class, I think you could give your constructor a method header hehe 👍